shiny/server.R

```diff
@@ -0,0 +1,55 @@
+library(shiny)
+##################################set work directory#########################
+setwd("/Users/panpancheng/Documents/study/capstone/shiny/crime_v1")
```

Comment:
This should probably be relative to the repo so that we can all run the code without modifying it (right now it is hardcoded to your username).